### ISSUE TITLE
ProcessTests.TestWorkingSet64 failed on OS X in CI

### ISSUE THREAD

**Issue by stephentoub** [dotnet/corefx#3281]
http://dotnet-ci.cloudapp.net/job/dotnet_corefx_mac_debug_tst/31/console
```
21:51:43    System.Diagnostics.ProcessTests.ProcessTests.TestWorkingSet64 [FAIL]
21:51:43       Assert+WrapperXunitException : File path: d:\j\workspace\dotnet_corefx_windows_debug\src\System.Diagnostics.Process\tests\ProcessTests.cs. Line: 339
21:51:43       ---- Assert.True() Failure
21:51:43       Expected: True
21:51:43       Actual:   False
21:51:43       Stack Trace:
21:51:43             at Assert.WrapException(Exception e, String callerFilePath, Int32 callerLineNumber)
21:51:43             at Assert.True(Boolean condition, String userMessage, String path, Int32 line)
21:51:43             at System.Diagnostics.ProcessTests.ProcessTests.TestWorkingSet64()
21:51:43          ----- Inner Stack Trace -----
21:51:43             at Assert.True(Boolean condition, String userMessage, String path, Int32 line)
```

**Comment by stephentoub** [comment]
Failed again here:
http://dotnet-ci.cloudapp.net/job/dotnet_corefx/job/osx_release_tst_prtest/13/console
```
21:17:33    System.Diagnostics.Tests.ProcessTests.TestWorkingSet64 [FAIL]
21:17:33       Assert+WrapperXunitException : File path: d:\j\workspace\dotnet_corefx\osx_release_bld_prtest\src\System.Diagnostics.Process\tests\ProcessTests.cs. Line: 337
21:17:33       ---- Assert.True() Failure
21:17:33       Expected: True
21:17:33       Actual:   False
21:17:33       Stack Trace:
21:17:33             at Assert.WrapException(Exception e, String callerFilePath, Int32 callerLineNumber)
21:17:33             at Assert.True(Boolean condition, String userMessage, String path, Int32 line)
21:17:33          ----- Inner Stack Trace -----
21:17:33             at Assert.True(Boolean condition, String userMessage, String path, Int32 line)
21:17:53 Finished:    System.Diagnostics.Process.Tests
```

**Comment by stephentoub** [comment]
Failed again here:
http://dotnet-ci.cloudapp.net/job/dotnet_corefx/job/osx_debug_tst_prtest/6/console

**Comment by stephentoub** [comment]
Failed again here:
http://dotnet-ci.cloudapp.net/job/dotnet_corefx/job/osx_release_tst_prtest/17/console

**Comment by Priya91** [comment]
fixed in #5732
